.gitlab-ci.yml

```diff
@@ -68,73 +68,78 @@ init_workspace:
     - chmod 700 ~/.ssh
     - ssh-keyscan github.com >> ~/.ssh/known_hosts
     # Clone all repos in WORKSPACE
+    - cd /tmp
+    - mv ${CI_PROJECT_DIR} /tmp/$(basename ${CI_PROJECT_DIR})
+    - rm -rf ${WORKSPACE}
+    - mkdir -p ${WORKSPACE}
+    - mv /tmp/$(basename ${CI_PROJECT_DIR}) ${CI_PROJECT_DIR}
```

Comment:
It took me some time to figure out what you were doing here. How about something like this, which is a bit clearer:

```
find ${WORKSPACE} \
    -maxdepth 1 \
    -not -name . \
    -not -name $(basename ${CI_PROJECT_DIR}) \
    -exec rm -rf '{}' ';'
```

Reply:
Thanks. Much cleaner...

Reply:
The `-not -name .` filter didn't work on my tests, I used `-mindepth 1` instead